verde/tests/test_utils.py

```diff
@@ -27,9 +27,22 @@
     meshgrid_to_1d,
     parse_engine,
     partition_by_sum,
+    fill_nans
 )
 
 
+def test_fill_nans():
+    """
+    This function tests the fill_nans function.
+    """
+    
+    grid = np.array([[1, np.nan, 3],
+                     [4, 5, np.nan],
+                     [np.nan, 7, 8]])
+    filled_grid = fill_nans(grid)
+    assert np.isnan(filled_grid).sum() == 0
```

Comment:
The function is supposed to take an `xarray.DataArray` but the test gives it a numpy array. The test should match the expected use of the function. Please make the grid into a `DataArray`.

It would also be good to check if the final grid has the correct values in the NaNs. Right now, this only checks that the NaNs aren't there but the values could be completely wrong and we'd never know.

Reply:
Hi Leo! 
A DataArray was added to the test function, and a check was made on the filled values in the DataArray. Could you please verify if the changes made are correct? Thank you for your attention.